server/channels/app/user.go

```diff
@@ -717,6 +791,8 @@ func (a *App) GetUsersByGroupChannelIds(c request.CTX, channelIDs []string, asAd
 	return usersByChannelId, nil
 }
 
+// GetUsersByUsernames returns a list of users based on the provided usernames, applying view restrictions if any.
```

Comment:
Maybe it'd be good to tell why there is an `asAdmin` flag, we can apply this to similar occurrences.